src/components/user/home.vue

```diff
@@ -46,26 +40,16 @@ const { tabPath, tabClass } = useTabs('/users');
 </script>
 
 <style lang="scss">
-@import '../../assets/scss/variables';
-
-#user-home .coming-soon {
-  color: $color-text;
-  font-size: 10px;
+#user-home {
+  #page-head-tabs li { display: block; }
```

Comment:
By default, `PageHome` hides the tab if it's the only one. Usually that's a useful thing, but here we do want to show the tab (see the discussion in the issue).